src/chttpd/src/chttpd.erl

```diff
@@ -1118,6 +1124,8 @@ error_info(all_workers_died) ->
         "Nodes are unable to service this "
         "request due to overloading or maintenance mode."
     >>};
+error_info({internal_server_error, Reason}) ->
+    {500, <<"internal server error">>, Reason};
```

Comment:
I wonder if maybe this _should_ have underscores, since most of the other seem to?